tests/test_connection_pool.py

```diff
@@ -525,6 +525,16 @@ def test_read_only_error(self, r):
         with pytest.raises(redis.ReadOnlyError):
             r.execute_command('DEBUG', 'ERROR', 'READONLY blah blah')
 
+    @skip_if_server_version_lt('2.8.8')
```

Comment:
Are you sure this was in 2.8.8? I think it's 2.8.21 based on the [git blame](https://github.com/redis/redis/commit/14ff572482cfe02c5ac61a1816cb8099d458e499) and tag release dates.